File: app/src/main/java/com/k4dima/chat/core/db/AppDatabase.java
package com.k4dima.chat.core.db;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import com.k4dima.chat.core.model.Message;

@Database(entities = {Message.class}, version = 1, exportSchema = false)
public abstract class AppDatabase extends RoomDatabase {
    public abstract MessageDao messageDao();
}

File: app/src/main/java/com/k4dima/chat/core/db/AppDatabase.kt
package com.k4dima.chat.core.db

import androidx.room.Database
import androidx.room.RoomDatabase

import com.k4dima.chat.core.model.Message

@Database(entities = [Message::class], version = 1, exportSchema = false)
abstract class AppDatabase : RoomDatabase() {
    abstract fun messageDao(): MessageDao
}